frame/balances/e2e-tests/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
I say we kiss and say goodbye to this altogether to keep the footprint minimal. I would add an example in the `node/e2e-test` that only demonstrates the capabilities of the testrunner: 

1. seal blocks
2. read state 
3. or anything else. 

This test per-se is not really meaningful.